modules/circe/build.sbt

```diff
@@ -1,11 +1,16 @@
 name := "pureconfig-circe"
 
-crossScalaVersions ~= { _.filterNot(_.startsWith("2.13")) }
+def onScala211(onScala211: String, onOthers: String) = Def.setting {
```

Comment:
Can we generalize this function a bit and move it to `project/`? It's a pretty good utility and we've been using it frequently, so I guess we'll need it again later in other places.

Reply:
Can we generalize it and move it if/when we need it? With #580 we no longer need to have different versions of ScalaCheck for different Scala versions so this becomes the only module requiring something like this. Project definitions tend to become bloated, so I'd rather keep the main `build.sbt` and whatever we have under the `project/` folder as simple as possible.

Reply:
I'm ok with adding this later 👍

It's more a question of when, not if. There are some projects that have a policy of maintaining only the latest two Scala versions (`http4s`, for which we also have a module, is one of them). In general, we can't reasonably expect that we'll always drop Scala versions before every other dependency we have.

I wouldn't find very bad to have general-purpose utilities like this in a `.scala` file in `project/`. Right now our build is fairly clean and because we already have a `Dependencies.scala` with it it wouldn't be any more bloated than what it is right now. Fun fact: I already added this function before for Scala 2.10, then removed it and later added the Scala 2.13 one we have now. We already went through this 🙂 

Accepting the PR to unblock this - feel free to merge if you still prefer to add it later :)

Reply:
Fair enough. I've added a `forScalaVersions` utility function in 3cc99c7. We can also use it in other locations of the build definition not related to dependencies.